microprofile/microprofile-rest-client-1.3/src/main/java/org/wildfly/swarm/ts/microprofile/rest/client/v13/sslsupport/SslClient.java

```diff
@@ -0,0 +1,13 @@
+package org.wildfly.swarm.ts.microprofile.rest.client.v13.sslsupport;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+@RegisterRestClient(configKey = "sslClient")
+public interface SslClient {
```

Comment:
The `CdiSslClient` and `SslClient` interfaces are identical, there's no need to have 2 of them. Remove the `CdiSslClient` thing, it's superfluous.